### PR TITLE
A few 1.16 mappings

### PR DESCRIPTION
**Pull request by Shnupbups**
Mapped new blocks and stuff

### CODE REVIEW

**Comment by liach** [comment]
@Shnupbups mind update to fix merge conflicts (on github)?

**Review by liach — APPROVED** [review]

**Comment by Shnupbups** [comment]
Mapped biome classes, added Shulker mapping from #1090

**Comment by Earthcomputer** [comment]
> Mapped biome classes

Careful not to overlap with other PRs ;)

File: mappings/net/minecraft/block/enums/WallType.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_4778 net/minecraft/block/enums/WallType
```
Comment (liach) [inline mappings/net/minecraft/block/enums/WallType.mapping]: More like wall shape/connection

**Review by Prospector — APPROVED** [review]

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/entity/ItemEntity.mapping [hunk 2]
```diff
@@ -23,6 +23,17 @@ CLASS net/minecraft/class_1542 net/minecraft/entity/ItemEntity
 		ARG 2 sourceEntity
 		ARG 3 sourceStack
 	METHOD method_20397 canMerge ()Z
+	METHOD method_24016 merge (Lnet/minecraft/class_1542;Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;)V
+		ARG 0 tergetEntity
```
Comment (Juuxel) [inline mappings/net/minecraft/entity/ItemEntity.mapping]:
```suggestion
		ARG 0 targetEntity
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Comment by liach** [comment]
I hope this is merged quickly. Current yarn mappings doesn't allow game bootstrap to run successfully as some Fungi block's constructor is protected and called by `Blocks`.

**Comment by Juuxel** [comment]
This should be ready once that typo is fixed.

**Review by Juuxel — APPROVED** [review]

**Pull request merged**